core/src/qos_service.rs

```diff
@@ -253,7 +253,7 @@ impl QosService {
             Ordering::Relaxed,
         );
         self.metrics.stats.estimated_execute_cu.fetch_add(
-            batched_transaction_details.costs.batched_execute_cost,
```

Comment:
It'd be nice to have a separate stat for the cost of builtin programs. So it would be:
1. estimated bpf compute
2. actual bpf compute
3. builtin compute

Reply:
good call, added